src/PointwiseFunctions/AnalyticData/GrMhd/CocalInitialData.cpp

```diff
@@ -0,0 +1,59 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "PointwiseFunctions/AnalyticData/GrMhd/CocalInitialData.hpp"
+
+#include <memory>
+#include <pup.h>
+
+namespace grmhd::AnalyticData {
+
+CocalInitialData::CocalInitialData(std::string data_directory,
+                                   const double electron_fraction,
```

Comment:
a bit strange that `electron_fraction` has to be input. Does the code actually do anything with it? Otherwise is there any generic code that can handle this (eg how is this done for an run-of-the mill polytropic TOV star)?